scripts/bash/help_script.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Command line for the script itself
+
+set -e
+
+echo "Enceladus Helper Scripts"
+echo ""
+echo "Usage: run_[job-name].sh [script-specific-options] [spark-specific-options] [job-specific-options]"
+echo ""
+echo "job-name:"
+echo "  standardization                 To run a Standardization only script"
+echo "  conformance                     To run a Conformance only script"
+echo "  standardization_conformance     To run a joint Standardization and Conformance script"
+echo ""
+echo "script-specific-options:"
+echo "  --help             To print this message"
+echo "  --asynchronous     To run the job in an asynchronous mode. The script will exit after printing application ID"
```

Comment:
Should be mentioned, that it works only in `cluster` mode (to be precise doesn't work in `client mode).